### PR TITLE
MISUV-792: fix tax year overview page H1 and title pattern accessibility issue…

### PR DESCRIPTION
**Pull request by scullythornton1**
…ity issue

### CODE REVIEW

File: app/views/templates/main_template.scala.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
                 timeout: 900,
                 countdown: 120,
                 keep_alive_url: "@controllers.timeout.routes.SessionTimeoutController.keepAlive().url",
-                title: "@messages("timeoutDialog.title")",
+                title: "@messages("timeoutDialog.title") + " " + @messages("titlePattern.serviceName.govUk")",
```
Comment (AlexRimmerHMRC) [inline app/views/templates/main_template.scala.html:43]: I don't think this is necessary, I think the "title" for the timeout is actually the heading of the popup that appears to users, in which case, we likely do not want the service name and gov.uk appended

File: app/views/templates/main_template.scala.html [hunk 2]
```diff
@@ -71,7 +71,7 @@
     @mainContent
 }
 
-@govuk_wrapper(title = title,
+@govuk_wrapper(title = title + " " + messages("titlePattern.serviceName.govUk"),
```
Comment (AlexRimmerHMRC) [inline app/views/templates/main_template.scala.html:74]: Not too much of a fan of this, might be cleaner to update the message to be
`{0} - Business Tax account - GOV.UK` that way here you could just do 
`title = messages("titlePattern.serviceName.govUk", title)`
without needing to use `+ " " +`

File: conf/messages.cy [hunk 3]
```diff
@@ -237,4 +237,7 @@ button.continue                                                 = Yn eich blaen
 
 ## Not found
 notFound.heading = Ni ellir dod o hyd i’r dudalen hon
-notFound.message = Gwiriwch eich bod wedi nodi’r cyfeiriad gwe cywir.
\ No newline at end of file
+notFound.message = Gwiriwch eich bod wedi nodi’r cyfeiriad gwe cywir.
+
+## Title pattern
+titlePattern.serviceName.govUk = - Cyfrif Treth Busnes - GOV.UK
```
Comment (AlexRimmerHMRC) [inline conf/messages.cy:243]: can you add a new line underneath this

File: app/views/templates/main_template.scala.html (hunk 1 above)
Comment (scullythornton1) [reply app/views/templates/main_template.scala.html:43]: done

File: app/views/templates/main_template.scala.html (hunk 2 above)
Comment (scullythornton1) [reply app/views/templates/main_template.scala.html:74]: done

**Pull request merged**